client/my-sites/email/email-management/home/email-plan-warnings.jsx

```diff
@@ -36,26 +31,17 @@ class EmailPlanWarnings extends React.Component {
 	renderCTAForTitanUnusedMailboxes() {
 		const { currentRoute, domain, selectedSiteSlug, translate } = this.props;
 
-		const showExternalControlPanelLink = ! isEnabled( 'titan/iframe-control-panel' );
-		const controlPanelUrl = showExternalControlPanelLink
-			? emailManagementTitanControlPanelRedirect( selectedSiteSlug, domain.name, currentRoute, {
-					context: TITAN_CONTROL_PANEL_CONTEXT_CREATE_EMAIL,
-			  } )
-			: emailManagementManageTitanAccount( selectedSiteSlug, domain.name, currentRoute, {
-					context: TITAN_CONTROL_PANEL_CONTEXT_CREATE_EMAIL,
-			  } );
+		const setupMailboxUrl = emailManagementTitanSetupMailbox(
+			selectedSiteSlug,
+			domain.name,
+			currentRoute
+		);
 
 		return (
-			<Button
-				compact
-				primary
-				href={ controlPanelUrl }
-				target={ showExternalControlPanelLink ? '_blank' : null }
-			>
+			<Button compact primary href={ setupMailboxUrl }>
 				{ translate( 'Activate Mailbox', 'Activate Mailboxes', {
 					count: getMaxTitanMailboxCount( domain ) - getConfiguredTitanMailboxCount( domain ),
 				} ) }
```

Comment:
Since a user can only set up one mailbox at a time, I wonder if we shouldn't display `Activate One Mailbox` instead of `Activate Mailboxes` when a user has several unused mailboxes:

<img width="471" alt="screenshot" src="https://user-images.githubusercontent.com/594356/127905424-337950a3-fdcc-4109-bd16-3b304bd17055.png">

Reply:
Actually maybe the CTA should be `Set Up Mailbox` and `Set Up A Mailbox` to better match the title and CTA from the `Set Up Mailbox` page:

<img width="475" alt="screenshot" src="https://user-images.githubusercontent.com/594356/127905813-5bfcb480-509e-4438-a928-9a34e2ac8047.png">

Reply:
You and I think alike @stephanethomas 😄 Just noticed your comments.